src/sass/base/_base.scss

```diff
@@ -22,13 +22,14 @@ time, mark, audio, video {
 }
 
 //TODO Implement this setting, fix current sizes before saving
-// html {
-//     // This defines what 1rem is
-//     font-size: 62.5%;
-// }
+html {
```

Comment:
It took me way too long to include this important feature in my base CSS. Lesson learned as I spend too much time fixing font sizes that should have been done this way the first time. **face palm**